docs/offboarding.rst

```diff
@@ -0,0 +1,162 @@
+Off-board Administrators and Journalists
+========================================
+
+When Journalists and SecureDrop Administrators leave your organization, it is important to off-board them from SecureDrop.
+
+**What you need:**
+
+- An *Admin Workstation*. Contact SecureDrop Support or follow our :doc:`guide to rebuilding an Admin Workstation <rebuild_admin>` if you do not have one.
+- An Admin account on the Journalist Interface
+
+.. important:: Note that, in particular for SecureDrop Administrators, additional measures may need to be considered if the user's departure is on unfavourable terms, and those measures will vary depending on the circumstances and your own internal incident response procedures. If you are in such a situation, consult the SecureDrop Support team or Digital Security Training team via the SecureDrop contact form, or email securedrop@freedom.press or a member of our team directly (found on the staff page). Note that, if you are in this situation, you may want to avoid reaching out to us via a ticket in our Support Portal, as that ticket will be visible to all Support Portal users at your organization.
+
+Off-boarding all users
+----------------------
+
+- `Inform the SecureDrop Support team`_ that the user's Support Portal account should be deactivated, and indicate if any new staff members should be added to the Support Portal..
+- De-activate the user's account on the Journalist Interface.
```

Comment:
Journalist Interface -> *Journalist Interface*

Reply:
Good point! Have changed to "delete" for now, but also welcome discussion re client behaviour. cc @creviera @redshiftzero